clientd3d/music.c

```diff
@@ -321,38 +393,64 @@ DWORD PlayMusicFile(HWND hWndNotify, const char *fname)
    DWORD dwReturn;
    MCI_PLAY_PARMS mciPlayParms;
    char temp[81];
-
-   if ((dwReturn = OpenMidiFile(fname)) != 0)
+   debug(( "Play music file %s. LatestMusic=%d, playing_music=%d, playing_midi=%d \n", 
+            fname, latest_music, playing_music, playing_midi ));
+   // If already playing music, pick up where we left off
+   if (isMusicPaused)
+   {
+      if (paused_music == bg_music && playing_music)
+      {
+         UnpauseMusic();
+         return 0;
+      }
+      else
```

Comment:
This doesn't need to be "else" since the "then" clause always returns.

Reply:
Thanks. Fixed.